client/app/visualizations/pivot/index.js

```diff
@@ -67,12 +67,12 @@ const PivotTableRenderer = {
 export default function init(ngModule) {
   ngModule.component('pivotTableRenderer', PivotTableRenderer);
 
-  ngModule.run(($injector) => {
+  ngModule.run(() => {
```

Comment:
You don't need `ngModule.run()` (and `ngModule` at all) now - it was needed only for two things: 1) register angular component (removed now); 2) retrieve `$injector` service - not needed as well. So just remove all this shit :slightly_smiling_face: